Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -1417,158 +1382,169 @@ def handle_newly_created_channel(creator, channel):
         return
 
 
-async def listen(client: SocketModeClient, req: SocketModeRequest):
-    """
-    This is the main listener which is attached to the open socket connection. When a SocketModeRequest has been received
-    this flow will be triggered. The Payload can be of any type and this function will determine what type it is, then
-    handle it accordingly. In the event that the request is nothing of interest to us, we will simply acknowledge the
-    request, process the message and gracefully ignore it.
-    :param client: SocketModeClient: This is the socket client which is created by the slack_loop function.
-    :param req: SocketModeRequest: The request object which has been sent by Slack.
-    :return: None
-    """
+def acknowledge_and_queue(client: SocketModeClient, req: SocketModeRequest):
     demisto.info("Handling request")
     if req.envelope_id:
         response = SocketModeResponse(envelope_id=req.envelope_id)
-        await client.send_socket_mode_response(response)
+        client.send_socket_mode_response(response)
```

Comment:
Maybe move this to after `MESSAGE_QUEUE.put(req)`, so we're sure we've got the message?
Also - You could do `client.send_socket_mode_response(response)` in another thread, as it's not important to your flow, and you don't need to wait for this network call.